scripts/meson-clang-common.cross-file

```diff
@@ -0,0 +1,2 @@
+[libarch:project options]
+install_headers = true
```

Comment:
Why is this called `clang-common`? Shouldn't this be something like `system-common` since it applies to `managarm-system`, not `managarm-kernel`?